packages/liveblocks-react/src/room.tsx

```diff
@@ -2819,8 +2819,54 @@ const _useThreadsSuspense: TypedBundle["suspense"]["useThreads"] =
  */
 const _useOther: TypedBundle["useOther"] = useOther;
 
-// TODO This one is tricky, as it has overloads
-const _useOthers: TypedBundle["useOthers"] = useOthers;
+/**
+ * Returns an object that lets you get information about all the users
```

Comment:
Was it written like this before? I find it a complicated way of describing an array.

Reply:
Ha, great point! Fixed in 203f5a584192934227c78862bc764f0a99ff7148. For context, Back In The Days™, this used to be an object that "proxied" an array (was array-like). But in 0.18 we changed it to an actual array.